### ISSUE TITLE
Updated to latest version...

### ISSUE THREAD

**Issue by MikeFalcor** [jakewaldron/PlexEmail#40]
So I recently updated PlexEmail to the latest version and now I am getting errors when I try to run it:

`Traceback (most recent call last):
  File "plexEmail.py", line 1037, in <module>
    imageInfo = processImage(hash, imageInfo['thumb'], 'show', 0, 0)
  File "plexEmail.py", line 343, in processImage
    imgName = thumb[thumb.index('_') + 1:len(thumb)]
ValueError: substring not found
`

I was able to modify previous errors away, but this one I cannot get my head around.  Any idea what could be causing this?

**Comment by BurntFrost** [comment]
Pull request 29 should fix this

https://github.com/jakewaldron/PlexEmail/pull/29

**Comment by MikeFalcor** [comment]
Alrighty.  So, I switched over to the dev build and manually added  the info from pull #29.

Here is the tail from the logging (nice job incorporating this, by the way!):
`2016-07-29 22:37:46,976 - INFO:processImage: Setting image paths to cloudinary
2016-07-29 22:37:46,980 - INFO:processImage: begin
2016-07-29 22:37:46,980 - INFO:processImage: imageHash = 42be2fb215fac6654b5b0a799de8811713a4d4f8 - thumb = https://thetvdb.com/banners/posters/305288-4.jpg - mediaType = show - seasonIndex = 0 - episodeIndex = 0
2016-07-29 22:37:46,981 - ERROR:Logging an uncaught exception
Traceback (most recent call last):
  File "plexEmail.py", line 1120, in <module>
    imageInfo = processImage(hash, imageInfo['thumb'], 'show', 0, 0)
  File "plexEmail.py", line 379, in processImage
    imgName = thumb[thumb.index('_') + 1:len(thumb)]
ValueError: substring not found
`

Bear in mind I have edited the HTML portion of plexEmail.py, so my lines are a little off.

It would appear as though the error is the same even after adding all of the pulls.  Any ideas?

**Comment by MikeFalcor** [comment]
Doh!  In the Dev version, you need to apply the pull bugfix about 5 times as opposed to only once in the master.  Testing again...

**Comment by MikeFalcor** [comment]
OR...I could actually pay attention to the pull and take note that it only applied to one specific section of code...

Don't mind me.  Added the pull correctly this time to the master version.  New error:
`title += movies[movie]['original_title'] + ' AKA '
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'`

Yay?

**Comment by MikeFalcor** [comment]
Actually...I will close this issue to keep things neat.  Will create new issue for new error.  Thanks!
